modules/button/react/lib/types.ts

```diff
@@ -7,17 +7,20 @@ export enum ButtonSizes {
   Large = 'large',
 }
 
-// TODO (beta button): consolidate all these button types
 /**
  * The different button types.
  */
-export enum ButtonTypes {
+
+/**
+ * @deprecated These type are deprecated along with deprecated_button component
```

Comment:
Nit: `deprecated_Button`

Reply:
We will have to add a writeup about the deprecation in the readme. Let's link to that from all of the `@deprecated` tags.

Reply:
Just following up, I think we can do this as a follow up PR